scripts/API.py

```diff
@@ -0,0 +1,438 @@
+# ***************************************************************************
+# * Authors:		Alberto García (alberto.garcia@cnb.csic.es)
+# *
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307 USA
+# *
+# * All comments concerning this program package may be sent to the
+# * e-mail address 'scipion@cnb.csic.es'
+# ***************************************************************************/
+
+"""
+Module containing all functions needed for the metric's API request.
+"""
+
+# General imports
+import re, hashlib, http.client, json
+
+# Self imports
+from typing import Optional, Tuple
+from utils import runJob
```

Comment:
Failing

Reply:
`runJob` is also part of the new installer. If this function did not get brought with the API functions, it is clearly going to fail.

Reply:
This runJob is part of the old installer too
https://github.com/I2PC/xmipp/blob/d51b92c7e7b8a772e26fdf9f0fef1a08640833a9/scripts/utils.py#L174